atd-vze/src/views/Crashes/Crash.js

```diff
@@ -292,7 +292,13 @@ function Crash(props) {
       )}
       <Row>
         <Col>
-          <CrashCollapses data={data} props={props} />
+          <CrashCollapses
+            data={crashData.crashes_by_pk}
+            refetch={refetch}
+            loading={loading}
+            error={error}
+            props={props}
```

Comment:
passing all of this down thru the components now so we can use just the one top level crash query